connect_ferc1_to_eia.py

```diff
@@ -16,101 +47,50 @@
-        'Unit Code': 'unit_id_pudl',
-        'EIA Technology': 'technology_description',
-        'Generator': 'generator_id',
-        'EIA Prime Mover': 'prime_mover_code',
-        'EIA Energy Source Code': 'energy_source_code_1',
-        'FERC Account': 'ferc_acct_name',
-        # use the RMI labels, not our eia_ownership relabel
-        'Owned or Total': 'ownership', }
-    string_cols = ['FERC Line Type', 'EIA Technology',
-                   'EIA Prime Mover', 'EIA Energy Source Code',
-                   'Owned or Total']
-    plant_part_rename = {'plant_part': {
-        'plant': 'plant',
-        'generator': 'plant_gen',
-        'unit': 'plant_unit',
-        'technology': 'plant_technology',
-        'plant_prime_fuel': 'plant_prime_fuel',
-        'plant_prime': 'plant_prime_mover'}, }
-
-    for col in string_cols:
-        if col in test_df.columns:
-            test_df.loc[:, col] = (
-                test_df[col].astype(str).
-                str.strip().
-                str.lower().
-                str.replace(r'\s+', '_')
-            )
-
-    test_df = (test_df.assign(report_date='2018-01-01').
-               rename(columns=cols_to_rename).
-               astype({'report_date': 'datetime64[ns]',
-                       'utility_id_eia': pd.Int64Dtype()}).
-               replace(plant_part_rename))
-
-    train_df_ids = compiler_mul.assign_record_id_eia(test_df)
-    train_df_ids['plant_part_og'] = train_df_ids['plant_part']
-    train_df_ids = (
-        train_df_ids.set_index(['record_id_eia']).
-        merge(compiler_mul.plant_parts_df[['true_gran', 'appro_part_label',
-                                           'appro_record_id_eia']],
-              how='left', right_index=True, left_index=True).
-        assign(plant_part=lambda x: x['appro_part_label'],
-               record_id_eia=lambda x: x['appro_record_id_eia']).
-        reset_index(drop=True))
-    # train_df_ids = compiler_mul.assign_record_id_eia(train_df_ids)
-
-    train_df_ids['record_id_ferc'] = (
-        train_df_ids.Source + '_' +
-        train_df_ids.report_year.astype(str) + '_' +
-        train_df_ids.report_prd.astype(str) + '_' +
-        train_df_ids.respondent_id.astype(str) + '_' +
-        train_df_ids.spplmnt_num.astype(str)
-    )
-    if "row_number" in train_df_ids.columns:
-        train_df_ids["record_id_ferc"] = train_df_ids["record_id_ferc"] + \
-            "_" + train_df_ids.row_number.astype('Int64').astype(str)
+def prep_train_connections(file_path_training, plant_parts_df):
+    """
+    Get and prepare the training connections.
+
+    We have stored connections between ferc1 steam and the eia master unit
+    list. These records should be compiled of connections between these data
+    sets that are known to be correct because we will use these records to
+    train a machine learning model.
+
+    Args:
+        training_file_path (path-like): path to the CSV of training data. The
```

Comment:
`file_path_training` in the function call signature vs. `training_file_path` here.